### DISCUSSION TITLE
New gateway

### DISCUSSION THREAD

**Discussion by pvuorio** [Koenkk/zigbee2mqtt#22832] [Devices]
Hello
Is there anyone that could help me to get the following device to work? Or point me to a place where i can get information.

Device according to label:
Zigbeee2mqtt GateWay
Model: GW_ZLU003P
Input: 5V=1A

It has USB-C and network port on the back.
Cant seem to be abe to communicate it, it identifies as a ch341-uart converter in linux

usb 1-1.3: New USB device found, idVendor=1a86, idProduct=7523, bcdDevice= 2.64
usb 1-1.3: New USB device strings: Mfr=0, Product=2, SerialNumber=0
usb 1-1.3: Product: USB Serial

But cant get zigbee2mqtt to start/recognice it

Regards,

Patrick

**Comment by francisp2** [comment]
Google on either GW_ZLU003P or ZLU003P does not give any results. Where did you get the device ?

**Reply by pvuorio** [reply]
Hi, I got it from Taobao when i was in china. maybe i just should scrap it and find something else.
